website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -4,7 +4,14 @@ id: "core"
 description: "Learn about semantic versioning for dbt Core, and how long those versions are supported."
 ---
 
-dbt Core releases follow [semantic versioning](https://semver.org/) guidelines. For more on how we use semantic versions, see [How dbt Core uses semantic versioning](#how-dbt-core-uses-semantic-versioning).
+dbt Core releases follow [semantic versioning](https://semver.org/) guidelines. For more on how we use semantic versions, see [How dbt Core uses semantic versioning](#how-dbt-core-uses-semantic-versioning). 
+
+dbt Labs provides different support levels to different versions, which may include new features, bug fixes, or security patches:
+
+- **[Active](#ongoing-patches)** &mdash; A few months after a minor version's initial release, we will provide bug fix releases to address any regressions and new bugs found in the initial release. 
+- **[Critical](#ongoing-patches)** &mdash; Newer minor versions transition the previous minor version into "Critical Support" with limited "security" releases for critical security and installation fixes.
+- **[End of Life](#eol-version-support)** &mdash; No longer receives bug fixes and limited to receiving only security patches. All EOL dbt Core v1.0 and later are available in dbt Cloud until further notice. 
+- **Deprecated** &mdash; Features in this state are not actively worked on or enhanced by dbt Labs.
```

Comment:
A few notes:
- "Active" support has become a bit more active than in the past. We will patch regressions, new bugs, and even include fixes for some older bugs / quality-of-life improvements, as long as we have high confidence that those changes are narrowly scoped and will not cause unintended side effects.
- "End of Life": Minor versions that have reached EOL do not get **any** new patch releases. If it's an issue with dependencies, we resolve this within dbt Cloud. We do not resolve it by putting out a new patch release. So this sentence is not correct: "limited to receiving only security patches."
- "Deprecated": We're not talking about features here, we're talking about older releases of dbt Core. Given that the current rule applies to all pre-1.0 (v0.X) versions, should we just say that explicitly here?
> **Deprecated:** dbt-core versions older than v1.0 are no longer maintained by dbt Labs, nor supported in dbt Cloud.

Reply:
Also: Should we snippetize this section, so it can be reused in the other page (without copy-paste)?

Reply:
thanks @jtcohen6 ! ok so to clarify the issue and [slack thread](https://dbt-labs.slack.com/archives/C0131TY7EEA/p1686693075613039?thread_ts=1686668217.878079&cid=C0131TY7EEA), is the below right:

> Minor versions that have reached EOL no longer receive new patch releases. If there's an issue with security or dependencies, we address this within dbt Cloud rather than through a new patch release. 


and yea ill create a snippet now. i thought it wasn't going to be much work since its only two places but best to change it in one place

Reply:
IMO let's just cut the second sentence, and leave this as:

> Minor versions that have reached EOL no longer receive new patch releases.

We continue to make those versions available in dbt Cloud. Occasionally, that requires us to pin certain upstream dependencies while building images for older versions within the dbt Cloud application. That's our implementation detail; I don't think that's something we need to document as part of this policy.

(& totally your call on whether the snippet is worth it or not!)